tests/sles4sap/netweaver_filesystems.pm

```diff
@@ -24,6 +24,14 @@ sub run {
     my $sap_dir = "/usr/sap/$sid";
     my ($proto, $path) = $self->fix_path(get_required_var('NW'));
     my $nfs_client_service_name = is_sle('16+') ? 'nfs-client.target' : 'nfs.service';
+    my $test_name = get_required_var('TEST');
+    $test_name =~ s/_(?:node\d+|supportserver)\z//i;
```

Comment:
I know you're not introducing here anything new regarding the test names, but perhaps a comment here regarding how the 3 jobs should be named (`$something_node01`, `$something_node02` and `$something_supportserver`)  could be helpful for the future.

Reply:
Agreed, on it